docs/operations.rst

```diff
@@ -0,0 +1,11 @@
+Camus Operations
+==========================
+
+In this document, you will learn how to use Camus in production.
+
+.. toctree::
+   :maxdepth: 3
+
+   monitoring
```

Comment:
@Ishiihara Have you thought about what could be useful to include in the deployment section? I would talk to @junrao and get in touch with current and past Camus maintainers to see if they ran into common issues and if they are willing to share any wisdom that we could include here.

It seems that we can probably get rid of the post deployment section or do you have anything useful to say here? Take a look at Kafka/schema registry to see examples.